csrc/scheduler/pointwise.cpp

```diff
@@ -1040,6 +1040,12 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
     auto output = entry.second;
     inner_most_tensors.erase(output);
   }
+  // IndexSelectOp supports vectorized load on lookupTv. It should be treated
+  // the same as a cached input and excluded from inner_most_tensors.
+  for (auto idx_sel : ir_utils::getOpsOfType<IndexSelectOp>(fusion)) {
+    inner_most_tensors.erase(idx_sel->output(0)->as<TensorView>());
```

Comment:
Should we do this only for vectorized IndexSelect ops?

Reply:
I think we should apply this blindly.

i.e. even without vectorization, aggregated allocation would still be beneficial to latency. I updated the comment to reflect that.